### PR TITLE
Our first Pull request

### PR DESCRIPTION
**Pull request by koerners**

### CODE REVIEW

**Comment by jbtrystram** [comment]
you should describe what your changes aim to do.

**Comment by jbtrystram** [comment]
And sync up with my repo instead of copy/pasting changes from another branch then re-commit

File: src/example/update/observation/StorageObserver.java [hunk 1]
```diff
@@ -61,8 +61,9 @@ public StorageObserver(String prefix) {
 
     // Control interface method. does the file handling
     public boolean execute() {
-
         StringBuilder log = new StringBuilder();
+        log.append("0"+  ";" + "1 \n");
+
```
[the remark this answers is not in the document]
Comment (koerners) [reply src/example/update/observation/StorageObserver.java]: Done

File: data_extraction.py [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+		neighbors_list = line.rstrip(';\n').split(';')
+		t.add_row([coord.iloc[A,0],coord.iloc[A,1],coord.iloc[A,2], storage.iloc[A, 1], progress.iloc[A,:], goodProgress(file_seq, path, A,1), goodProgress(file_seq, path, A, 2), neighbors_list])
+		A+=1
+
+		
+
+	print(t.draw())
+	f.write(t.draw())
+
```
[the remark this answers is not in the document]
Comment (koerners) [reply data_extraction.py]: Cleaned up

File: README.md [hunk 3]
```diff
@@ -1,32 +0,0 @@
-# Discovery-based P2P data distribution
```
[the remark this answers is not in the document]
Comment (koerners) [reply README.md]: Was unintentionally

File: .gitignore [hunk 4]
```diff
@@ -9,3 +9,7 @@ build.xml
 
 figs/
 raw_dat/
+
+venv/bin/
+
```
Comment (jbtrystram) [inline .gitignore]: you really have something with empty lines don't you ?

File: src/example/update/constraints/NodeDetails.java [hunk 5]
```diff
@@ -1,6 +1,6 @@
 package example.update.constraints;
 
-import example.update.initialisation.InetInitializer;
+import example.update.initialisation.CoordinatesInitializer;
```
Comment (jbtrystram) [inline src/example/update/constraints/NodeDetails.java]: do a separate PR with this!

Comment (koerners) [reply src/example/update/constraints/NodeDetails.java]: Tells me something along the lines of "A pull request for ...  already exists" if I try to make a new one

Comment (koerners) [reply src/example/update/constraints/NodeDetails.java]: Tells me something along the lines of "A pull request for ...  already exists" if I try to make a new one

Comment (koerners) [reply src/example/update/constraints/NodeDetails.java]: Tells me something along the lines of "A pull request for ...  already exists" if I try to make a new one
